src/main/java/com/lorepo/icplayer/public/libs/player-utils.js

```diff
@@ -87,7 +105,25 @@
                         "mistake_count": score['mistakeCount'] ? score['mistakeCount'] : 0,
                         "time": score['time'] ? parseInt(score['time'], 10) : 0
                     };
-
+                    if (includeNonReportable) {
+                        paginatedResults[count]["reportable"] = true;
+                    }
+                    count += 1;
+                } else if (includeNonReportable) {
+                    var pageTime = this.timeService.getPageTimeById(page.getId());
+                    paginatedResults[count] = {
+                        "page_number": (i + 1),
+                        "page_name": page.getName(),
+                        "page_id": page.getId(),
+                        "score": 0.0,
+                        "absolute_score": 0,
+                        "max_score": 0,
+                        "errors_count": 0,
+                        "checks_count": 0,
+                        "mistake_count": 0,
+                        "time": pageTime ? parseInt(pageTime, 10) : 0,
+                        "reportable": false
+                    };
                     count += 1;
```

Comment:
count jest używany także jako maxScore. `maxScore: count` w dict na końcu funkcji. Skoro to nieraportowane to chyba nie powinny się wliczać.

Reply:
Ciekawe, jest to że sumOfMaxScore nie jest nigdzie wykorzystywane.
Jednak biorąc pod uwagę, że `rawScore: sumOfScaledScore`, zaś 
`if (sumOfWeights) {
    scaledScore = Math.round((sumOfScaledScore / sumOfWeights) * 100) / 100;
} else {
    scaledScore = 1;
 }` to ma sens, że maxScore nie powinien wliczać nieraportowalnych, gdyż nieraportowane nie zwiększają liczników powyższych zmiennych